Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -360,6 +374,12 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerror: true
```

Comment:
Why this is needed?

Reply:
same as the !cve command - the error config is set to 'continue' if an error occurs since the API returns an API error even when the results are returned.

Reply:
Sound like a bug